### PR TITLE
chore: upgrade lerna to v5 latest

### PR DESCRIPTION
**Pull request by gbockus-sf**
### What does this PR do?
Upgrade lerna to version 5 and made a few associated changes to keep everything building happily

### What issues does this PR fix or reference?
@W-11384712@

### Functionality Before
We would hit the MAX_BUFFER error regular when running the lint task locally 

### Functionality After
No more MAX_BUFFER issue
We have the option to start using NX

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@commitlint/cli": "^15.0.0",
     "@commitlint/config-conventional": "^15.0.0",
     "@monorepo-utils/workspaces-to-typescript-project-references": "2.7.6",
-    "@octokit/core": "3.6.0",
+    "@octokit/core": "4.0.4",
```
Comment (gbockus-sf) [inline package.json:19]: required to make peer deps happy post lerna upgrade

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "husky": "1.3.1",
     "istanbul": "1.1.0-alpha.1",
     "jsdom": "^15.1.1",
-    "lerna": "3.22.1",
+    "lerna": "5.4.3",
```
Comment (gbockus-sf) [inline package.json:33]: lerna latest

File: packages/salesforcedx-sobjects-faux-generator/package.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "compile": "webpack",
     "webpack": "webpack",
-    "lint": "tslint --project . -e **/src/data/minSObjects.json",
+    "lint": "tslint --project . -e **/minSObjects.json",
```
Comment (gbockus-sf) [inline packages/salesforcedx-sobjects-faux-generator/package.json:37]: not sure what was fixed in lerna to make this start failing post upgrade, but :fixed:

File: packages/salesforcedx-utils-vscode/package.json [hunk 4]
```diff
@@ -50,9 +50,9 @@
     "lint:fix": "npm run lint -- --fix",
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
-    "test": "npm run test:unit",
-    "test:unit": "node ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/unit/**/*.js --timeout 20000 --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
-    "coverage": "./node_modules/.bin/nyc npm test"
+    "test:unit": "mocha --recursive out/test/unit/**/*.js --timeout 20000 --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
+    "test:unit:coverage": "node ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/unit/**/*.js --timeout 20000 --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
+    "coverage": "./node_modules/.bin/nyc npm test:unit:coverage"
```
Comment (gbockus-sf) [inline packages/salesforcedx-utils-vscode/package.json:55]: So with the new version of lerna this was always encountering an error when using nyc to generate coverage number on utils due to some webpack issue. I just removed the coverage generation for the default test:unit task and added a specific script for having coverage locally. Note this was only occurring on the windows build.

**Review by klewis-sfdc — APPROVED** [review]

**Pull request merged**